webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/Boot.scala

```diff
@@ -244,7 +244,7 @@ class Boot extends Loggable {
 
     //exclude Rudder doc from context-path rewriting
     LiftRules.excludePathFromContextPathRewriting.default.set(() => (path:String) => {
-      val noRedirectPaths= "/rudder-doc" :: "/ncf" :: "/ncf-builder" :: Nil
+      val noRedirectPaths= "/rudder-doc" :: Nil
```

Comment:
I think that's a bug and should be removed for all 7.x branches

Reply:
It is a leftover that does not exist anymore, not really a bug per se IMHO